CONTRIBUTING.rst

```diff
@@ -0,0 +1,159 @@
+Contributing to the Jupyter Notebook
+====================================
+
+TODO: a welcoming sentence
+
+General Guidelines
+------------------
+
+For general documentation about contributing to Jupyter projects, see the
+`Project Jupyter Contributor Documentation`__.
+
+__ http://jupyter.readthedocs.org/#contributor-documentation
+
+
+Setting Up a Development Environment
+------------------------------------
+
+For general installation instructions have a look at the `Project Jupyter Installation Guide`__.
+
+__ https://jupyter.readthedocs.org/en/latest/install.html
+
+Installing Node.js and npm
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Building the Notebook from its GitHub source code requires some tools to
+create and minify JavaScript components and the CSS.
+
+You can use the pre-built installer from the `Node.js website`__.
+The installer will include Node.js and Node's package manager, ``npm``.
+
+__ https://nodejs.org
+
+Or you can use your system's package manager ...
+
+If you use Homebrew_ on Mac OS X::
+
+    brew install node
+
+.. _Homebrew: http://brew.sh/
+
+For Debian/Ubuntu systems, you should use the ``nodejs-legacy`` package instead
+of the ``node`` package::
+
+    sudo apt-get update
+    sudo apt-get install nodejs-legacy npm
+
+
```

Comment:
I moved a few things around and added the conda command: 3c1fcedfaac61693c7ec470b377b04a7caf6c72f

Now the conda command is the first suggestion, then come the other package managers and last (and I suppose least, too) comes the installer from the website.

It's very easy to add further package managers in the future, if that's necessary.